contributing.md

```diff
@@ -22,38 +18,89 @@ $ cd microsoft-authentication-library-for-js
-or the master branch.  **All bug fixes and new features should go into the dev branch.**
-
-The master branch is effectively frozen; patches that change the SDKs
-protocols or API surface area or affect the run-time behavior of the SDK will be rejected.
+Now decide if you want your feature or bug fix to go into the dev branch or the master branch.  **All bug fixes and new features should go into the dev branch.**
 
-Some of our SDKs have bundled dependencies that are not part of the project proper.  Any changes to files in those directories or its subdirectories should be sent to their respective
-projects.  Do not send your patch to us, we cannot accept it.
+The master branch is effectively frozen; patches that change the SDKs protocols or API surface area or affect the run-time behavior of the SDK will be rejected.
 
-In case of doubt, open an issue in the [issue tracker][].
+Some of our SDKs have bundled dependencies that are not part of the project proper.  Any changes to files in those directories or its subdirectories should be sent to their respective projects. Do not send your patch to us, we cannot accept it.
 
-Especially do so if you plan to work on a major change in functionality.  Nothing is more
-frustrating than seeing your hard work go to waste because your vision
-does not align with our goals for the SDK.
+In case of doubt, open an [issue](https://github.com/AzureAD/microsoft-authentication-library-for-js/issues).
 
+Especially do so if you plan to work on a major change in functionality.  Nothing is more frustrating than seeing your hard work go to waste because your vision does not align with our goals for the SDK.
 
-### BRANCH
+### Create a branch for your feature/patch
 
-Okay, so you have decided on the proper branch.  Create a feature branch
-and start hacking:
+Okay, so you have decided on the proper branch.  Create a feature branch and start hacking:
 
 ```
 $ git checkout -b my-feature-branch
 ```
 
-### BOOTSTRAP
+## Installation and Bootstrapping
+
+This repository is a monorepo for all packages maintained by the MSAL.js team. It contains the following public npm packages:
+
+- [msal](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-core)
+- [@azure/msal](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-core) (same package as above)
+- [@azure/msal-angular](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-angular)
+- [@azure/msal-common](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-common)
+- [@azure/msal-browser](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-browser)
+- [@azure/msal-node](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-node)
+- [@azure/msal-node-extensions](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/extensions/msal-node-extensions)
```

Comment:
I might just link to the `lib` folder, so we don't have to keep this up to date.

Reply:
updated. Need to link to lib and extensions.